src/mondeath.cpp

```diff
@@ -648,15 +648,25 @@ void mdeath::broken( monster &z )
     }
     // make "broken_manhack", or "broken_eyebot", ...
     item_id.insert( 0, "broken_" );
-
```

Comment:
Why'd you remove this line?

Reply:
You mean the carriage return? Maybe a nervous tick. It's my first significant change (as opposed to balancing json) so I forgot to pay attention to new lines. Sometimes I added/deleted a bunch just to help me see where my code was.